src/Build/project.json

```diff
@@ -14,19 +16,26 @@
     "netstandard1.5": {
       "dependencies": {
         "NETStandard.Library": "1.6.0",
+        "NuGet.Commands": "4.5.0",
+        "NuGet.Protocol": "4.5.0",
```

Comment:
nit: since these are the same across frameworks it'd be nice to have them in a common node.

Reply:
Okay